advanced/am-pattern-4/Chart.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+appVersion: "3.1.0"
+description: A Helm chart for the deployment of WSO2 API Management deployment pattern 2
+name: am-pattern-4
+version: 3.1.0-3
```

Comment:
@chathuraswso2 the next release will be `3.1.0-4`. Please update.

Also, update in other Helm charts as well.

Reply:
https://github.com/wso2/kubernetes-apim/pull/444/commits/088623390e99bf605cb653d258ee715430d7043d